src/models/sheltersModel.js

```diff
@@ -13,6 +14,27 @@ const getAllShelters = async (params) => {
     });
 }
 
+const createShelters = async (params) => {
+
+    var vals = `${SqlUtil.Quote(params.shelterName)}, ${SqlUtil.Quote(params.address)}, ${SqlUtil.Quote(params.emailAddress)},
+     ${SqlUtil.Quote(params.password)}, ${SqlUtil.Quote(params.phoneNumber)}, ${SqlUtil.Quote(params.website)}, ${SqlUtil.Quote(new Date())}`
+
+    const sql = `INSERT INTO Shelter (ShelterName, Address, EmailAddress, Password, PhoneNumber,Website,LastUpdated) 
+    VALUES (${vals})`;
+    //console.log("Running insert query=" + sql )
+
+    return new Promise((resolve, reject) => {
+      db.query(sql, (err, res, fields) => {
+          if (err) {
+              reject(err);
+          } else {
+              resolve(res);
+          }
+      });
+    });
+}
+
 module.exports = {
-	getAllShelters
+	getAllShelters,
+    createShelters
```

Comment:
Ugly! I will the indentation with next PR.

Reply:
Since I'm requesting changes, I think this can be updated in the next commit for this PR if you don't mind

Reply:
fixed